docs/developer/hacking.rst

```diff
@@ -0,0 +1,51 @@
+#######
+Hacking
+#######
+
+.. note:: Before reading this file, please read the :doc:`kytos:contributing`
+    section that contains the main guidelines of the project.
+
+Development Environment Setup
+*****************************
+
+It is possible to install any of the *kytos* repos in develop mode, so that the
+installed files are kept in the original location, and only links are made to
+the system locations. This allows developpers to quickly test changes made in
```

Comment:
@erickvermot Sorry, I didn't see this type on the first review.
`s/developpers/developers`.